types/jwk-to-pem/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for jwk-to-pem 2.0
+// Project: https://github.com/Brightspace/node-jwk-to-pem#readme
+// Definitions by: Erik Silkensen <https://github.com/esilkensen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = jwkToBuffer;
```

Comment:
I don't know whether it matters or one name would be better than the other - `jwkToBuffer` vs. `jwkToPem`? (`jwkToBuffer` matches the source definition)

Reply:
I think matching the source definition is best here. It doesn't really matter, since the name you use here is not externally visible.